static/src/stylesheets/amp/_geo.scss

```diff
@@ -8,3 +8,10 @@
 .amp-geo-no-group .geo-amp-ad--row {
     display: block;
 }
+
+.amp-geo-group-us .liveblog-geo-amp-ad--us,
```

Comment:
Would it make sense to decouple the styling from the region? I guess this means if we add new regions we'll have to remember to update these rules to have it display properly.

Reply:
Unfortunately the hack depends on the styling.  This was Google's suggestion for how to have different configuration for each ad region based on geolocation.  At the moment you can't just substitute a variable according to some context like geolocation; you have to hide or show an entire element that contains its own configuration.  When you hide an `amp-ad` element it doesn't make any calls out from its `rtc-config` attribute until it becomes visible.
The description of #20840 explains how it works.